src/apps/filters/h3.c

```diff
@@ -1632,6 +1632,811 @@ SUBCOMMAND(cellsToMultiPolygon,
+}
+
+SUBCOMMAND(cellToVertexes,
+           "Returns all of the vertexes from the specified cell") {
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&cellToVertexesArg, &cellArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    H3Index out[6] = {0};
+    // This one is pretty loose about the inputs it accepts, so let's validate
+    // for it
+    bool isValid = H3_EXPORT(isValidCell)(cell);
+    if (!isValid) {
+        return E_CELL_INVALID;
+    }
+    H3Error err = H3_EXPORT(cellToVertexes)(cell, &out[0]);
+    if (err != E_SUCCESS) {
+        return err;
+    }
+    printf("[");
+    bool hasPrinted = false;
+    for (int i = 0; i < 6; i++) {
+        if (out[i] > 0) {
+            if (hasPrinted) {
+                printf(", ");
+            }
+            printf("%" PRIx64, out[i]);
+            hasPrinted = true;
+        }
+    }
+    printf("]\n");
+    return E_SUCCESS;
+}
+
+SUBCOMMAND(vertexToLatLng, "Returns the lat, lng pair for the given vertex") {
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&vertexToLatLngArg, &cellArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    bool isValid = H3_EXPORT(isValidVertex)(cell);
+    if (!isValid) {
+        return E_VERTEX_INVALID;
+    }
+    LatLng ll;
+    H3Error err = H3_EXPORT(vertexToLatLng)(cell, &ll);
+    if (err) {
+        return err;
+    }
+    // Using WKT formatting for the output. TODO: Add support for JSON
+    // formatting
+    printf("POINT(%.10lf %.10lf)\n", H3_EXPORT(radsToDegs)(ll.lng),
+           H3_EXPORT(radsToDegs)(ll.lat));
+    return E_SUCCESS;
+}
+
+SUBCOMMAND(isValidVertex,
+           "Checks if the provided H3 vertex is actually valid") {
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&isValidVertexArg, &helpArg, &cellArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    bool isValid = H3_EXPORT(isValidVertex)(cell);
+    printf("%s", isValid ? "true\n" : "false\n");
+    return E_SUCCESS;
+}
+
+/// Miscellaneous subcommands
+
+SUBCOMMAND(degsToRads, "Converts degrees to radians") {
+    double deg = 0;
+    Arg degArg = {.names = {"-d", "--degree"},
+                  .required = true,
+                  .scanFormat = "%lf",
+                  .valueName = "DEG",
+                  .value = &deg,
+                  .helpText = "Angle in degrees"};
+    Arg *args[] = {&degsToRadsArg, &degArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    printf("%.10lf\n", H3_EXPORT(degsToRads)(deg));
+    return E_SUCCESS;
+}
+
+SUBCOMMAND(radsToDegs, "Converts radians to degrees") {
+    double rad = 0;
+    Arg radArg = {.names = {"-r", "--radian"},
+                  .required = true,
+                  .scanFormat = "%lf",
+                  .valueName = "RAD",
+                  .value = &rad,
+                  .helpText = "Angle in radians"};
+    Arg *args[] = {&radsToDegsArg, &radArg, &helpArg};
+    PARSE_SUBCOMMAND(argc, argv, args);
+    printf("%.10lf\n", H3_EXPORT(radsToDegs)(rad));
```

Comment:
Nit: Worth making a common `printDouble` function to help us always print with the same format?